src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -190,11 +191,11 @@ protected override string GenerateFullPathToTool()
         {
             string pathToTool = ToolLocationHelper.GetPathToBuildToolsFile(ToolName, ToolLocationHelper.CurrentToolsVersion);
 
-            if (null == pathToTool)
+            if (pathToTool == null)
```

Comment:
This is a habit from C/C++, but it it's not a bad habit to have. Putting the constant on the left is called a "Yoda conditional" and in languages like C/C++ that don't present warnings on assignments in conditionals this prevents you from accidentally using `=` instead of `==`. It's not necessary in C#, but I think a lot of people will want to use the same style and since it isn't wrong to do so in C# I don't think we should go about changing it.